flintrock/scripts/install-spark.sh

```diff
@@ -2,22 +2,20 @@
 
```

Comment:
This is an unfortunately inconsistency, and eventually I think both scripts should be in Python. The reason `download-hadoop` was written in Python was because of the Apache mirror selection logic, which seemed like a bit much to do purely in Bash.